components/atoms/PullRequestOverviewChart/pull-request-overview-chart.tsx

```diff
@@ -10,24 +10,29 @@ const PullRequestOverviewChartBar: React.FC<PullRequestOverviewChartBarProps & R
   type,
   ...event
 }) => {
+  const getBarColor = (type: string) => {
+    switch (type) {
+      case "open":
+        return "bg-light-grass-9";
+      case "merged":
+        return "bg-purple-600";
+      case "closed":
+        return "bg-light-red-9";
+      case "draft":
+        return "bg-light-slate-9";
+      default:
+        return "";
+    }
+  };
+
   return (
     <div
       {...event}
       className={`
-        ${
-          type === "open"
-            ? "bg-light-grass-9"
-            : type === "merged"
-            ? "bg-purple-600"
-            : type === "closed"
-            ? "bg-light-red-9"
-            : type === "draft"
-            ? "bg-light-slate-9"
-            : ""
-        }
-    ${percent === 0 && "hidden"}
+        ${getBarColor(type!)}
+        ${percent === 0 ? "hidden" : "block"}
         transition-all shrink-0 duration-500 ease-in-out rounded-full`}
-      style={{ width: `${percent}%` }}
+      style={{ width: `${Math.round(percent!)}%` }}
```

Comment:
This appears to be the actual fix?

Reply:
This is just a small improvement to remove decimal points from the percent in the markup styles

Reply:
If the intent is to have all lines add up to 100%, there are edge cases here that won't be accounted for. For example:

```js
console.log(
    Math.round(50.01),
    Math.round(49.99 / 3),
    Math.round(49.99 / 3),
    Math.round(49.99 / 3)
);
// 50 17 17 17
```

which adds up to 101.

Also, this will drop any precision on 0 below `0.5` which will cause those parts to not render:

```js
console.log(Math.round(0.4));
// 0
```

---

We may want to consider adding additional precision onto these calculations if we care about having all parts render (unless it is truly 0) and we want it to all add up to 100

Reply:
> This is just a small improvement to remove decimal points from the percent in the markup styles

Makes sense. And if we're going for more or less "close" to adding up to 100 and we're ok dropping percents below `0.4`, then we'll just want to be aware of these edge cases or account for them if that's unacceptable from the design perspective.

Reply:
reverted the change for now since the previous one broke nothing so we can move on. will revisit another time .

Thanks for the observations 👍